index.html

```diff
@@ -837,14 +837,12 @@ <h1>Introduction</h1>
 
 
  
- <aside class="example" id="thing-description-full-serialization" title="Thing Description with context extension">
+ <aside class="example" id="thing-description-full-serialization" title="Thing Description with Context Extension for semantic annotations">
```

Comment:
Capitalization should be consistent: Thing Description with Context Extension for Semantic Annotations.   Or was Context Extension capitalized because it was is a special term, now?

Reply:
Defined terms capitalized